WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/RegistrationWizard.java

```diff
@@ -0,0 +1,76 @@
+package nl.tudelft.watchdog.ui.wizards;
+
+import nl.tudelft.watchdog.ui.preferences.Preferences;
+import nl.tudelft.watchdog.ui.util.UIUtils;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectCreatedEndingPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectIdEnteredEndingPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectRegistrationPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectSliderPage;
+import nl.tudelft.watchdog.ui.wizards.projectregistration.ProjectWelcomePage;
+
+import org.eclipse.jface.wizard.Wizard;
+
+/** Parent class for User and Project registration wizards. */
+public abstract class RegistrationWizard extends Wizard {
+
+	/**
+	 * The project-id, either entered on the previous wizard pages or as
+	 * retrieved by the server.
+	 */
+	protected String projectId;
+
+	/** Maximum number of pages in the current wizard. */
+	protected int totalPageNumber;
+
+	/** Page with Yes/No question about registering new project ID. */
+	public ProjectWelcomePage projectWelcomePage;
```

Comment:
My previous comment refers to your first comment. For the second, it is arguable - at the moment, these pages are part of both wizards, so it is justified to leave them in a base class. On the other hand, moving them downwards (to subclasses) would be okay if we have some more registrations, but I am guessing it will not happen in the near future. 
My reason for putting these pages there is to be able to access them from (RegistrationWizard) getWizard()